apps/screener-proxy/types/github.ts

```diff
@@ -0,0 +1,27 @@
+export interface GithubWebhook {
```

Comment:
There should be actual types you can use for this from `@octokit/types` (use the same version from the scripts package)

Reply:
Added `@octokit/types` and `@octokit/webhooks-definitions` to replace the custom interfaces